server/command_hooks.go

```diff
@@ -88,6 +88,35 @@ func (p *Plugin) executeCommandExport(args *model.CommandArgs) *model.CommandRes
 		}
 	}
 
+	var channelToExportName, exportSuccessMsg, exportProcessMsg string
+	if channelToExport.Type == model.ChannelTypeOpen || channelToExport.Type == model.ChannelTypePrivate {
+		channelToExportName = channelToExport.Name
+		exportSuccessMsg = fmt.Sprintf("Channel ~%s exported:", channelToExportName)
+		exportProcessMsg = fmt.Sprintf("Exporting ~%s. @%s will send you a direct message when the export is ready.", channelToExportName, botUsername)
+	} else if channelToExport.Type == model.ChannelTypeGroup {
+		channelToExportName = channelToExport.Name
+		team, err := p.API.GetTeam(args.TeamId)
+		if err != nil {
+			p.client.Log.Error("error occurred while getting the details of team for the channel.",
+				"GM channel ID", args.ChannelId, "User ID", args.UserId, "Error", err)
+
+			return &model.CommandResponse{
+				ResponseType: model.CommandResponseTypeEphemeral,
+				Text:         fmt.Sprintf("An error occurred trying to exporting the chat for %s GM channel", channelToExportName),
+			}
```

Comment:
Having this link is not a must so we can just log the error and continue with our functionality without a link